integration_test_lite.py

```diff
@@ -1,50 +1,80 @@
-import requests
+import asyncio
 import json
 import sys
 import uuid
 
-
-# Helper function to pretty print a list or dictionary
-def pretty_print(print_me):
-    print(json.dumps(print_me, indent=4, sort_keys=True))
-
-
-# Helper function to get an access token
-def get_access_token():
-    post_data = {
-        'client_id': config_json["client_id"],
-        'client_secret': config_json["client_secret"],
-        'grant_type': 'client_credentials'
-    }
-    request = requests.post(config_json["token_api"], data=post_data)
-
-    return f'Bearer {request.json()["access_token"]}'
+import aiohttp
 
 
-# Basic http request with url and query parameters
-def basic_request(url, params, needs_access_token, verb="get"):
-    headers = {}
-
-    if (needs_access_token):
-        headers['Authorization'] = access_token
-
-    request = requests.request(verb, url, params=params, headers=headers)
-    return request
+def pretty_print(print_me):
+    """Helper function to pretty print a list or dictionary
 
+    :param print_me: The list/dictionary to be printed
+    """
+    print(json.dumps(print_me, indent=4, sort_keys=True))
 
-# Makes a call to each API and adds the API
-# to a list if the call wasn't successful
-def get_bad_apis():
-    bad_apis = []
 
-    for endpoint in config_json["target_endpoints"]:
+class IntegrationTestLite:
+    def __init__(self):
+        config_data_file = open(sys.argv[1])
+        self.config_json = json.load(config_data_file)
+        self.session = aiohttp.ClientSession()
+
+    async def __aenter__(self):
+        return self
+
+    async def __aexit__(self, *excinfo):
+        await self.session.close()
+
+    async def set_access_token(self):
+        """Helper function to set the access token"""
+        session = self.session
+        config_json = self.config_json
+        post_data = {
+            'client_id': config_json["client_id"],
```

Comment:
`"client_id"` should be wrapped with single-quotes. There're a few in other places.